src/pages/SignUp/SignUp.js

```diff
@@ -1,7 +1,151 @@
-import React from "react";
+import React, { useState } from "react";
+import FindAddr from "../../components/SingUp/FindAdrr";
+import "./SignUp.scss";
 
 const SignUp = () => {
-  return <div>signup</div>;
+  const [userInfo, setUserInfo] = useState({
+    id: "",
+    pw: "",
+    pwCheck: "",
+    name: "",
+    phone: "",
+    address: "",
+    birth: "",
+  });
+
+  const [daumAddress, setDaumAddress] = useState(false);
+
+  const [valueCheck, setValueCheck] = useState(false);
+
+  const linkToDaum = () => {
+    setDaumAddress(!daumAddress);
+  };
+
+  const putUserInfo = e => {
+    setUserInfo(prev => ({ ...prev, [e.target.name]: e.target.value }));
+    console.log(userInfo);
+  };
+  // console.log(valueCheck, "valueCheck");
+
+  const alertText = () => {
+    setValueCheck(userInfo.id ? true : false);
+  };
```

Comment:
```suggestion
  const showAlert = () => {
    setValueCheck(userInfo.id ? true : false);
  };
```
* alert 보여주는 기능을 하는 함수 이기 때문에, 함수는 동사형으로 이름 지어주는게 좋습니다.
* 메시지가 최초에 focus일때는 안보였다가, onBlur 되면 true상태로 남아있게 하면 되겠네요